osm2gtfs/creators/esteli/esteli.json

```diff
@@ -0,0 +1,35 @@
+{
+    "query": {
+        "bbox": {
+            "n": "13.1308234",
+            "s": "13.0534171",
+            "e": "-86.3197351",
+            "w": "-86.3999775"
+        },
+        "tags": {
+            "route": "bus",
+            "network": "NI-Estelí"
+        }
+    },
+    "stops": {
+        "name_without": "Parada sin nombre",
+        "name_auto": "no"
```

Comment:
Shouldn't we use this function?

Reply:
I don't know - do you want to use it? In Managua we abstain, as we have all stops with names.

Reply:
All the stops have now names in Esteli, too.

Reply:
If we have all stop names, no, it does not make sense.